src/components/guide/OnboardingModal.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <Modal
+    :heading="fluent(`onboarding_modal_${step}`)"
+    :initiallyOpen="true"
+    :closeButton="true"
+    @close="done"
+  >
+    <div class="onboarding-modal__dots">
+      <ProgressDots :dots="steps.length" :progress="step" />
+    </div>
+    <article class="onboarding-modal__article">
+      <img :src="step_data.img" />
+      <p>
+        <Fluent
+          :id="`onboarding_modal_${step}`"
+          attr="paragraph_1"
+          :tags="step_data.tags"
+        />
+      </p>
+      <p>
+        <Fluent
+          :id="`onboarding_modal_${step}`"
+          attr="paragraph_2"
+          :tags="step_data.tags"
+        />
+      </p>
+    </article>
+    <hr class="onboarding-modal__hline" />
+    <footer class="onboarding-modal__footer">
+      <a href="#" @click="prev()" :class="[{ 'button--hidden': first }]">{{
+        fluent('back')
+      }}</a>
+      <a href="#" class="button" @click="last ? done() : next()">{{
+        fluent(last ? 'close' : 'next')
+      }}</a>
+      <a href="#" @click="done()" :class="[{ 'button--hidden': last }]">{{
+        fluent('skip')
+      }}</a>
+    </footer>
+  </Modal>
+</template>
+
+<script>
+import Fluent from '@/components/Fluent.vue';
+import Modal from '@/components/_functional/Modal.vue';
+import ProgressDots from '@/components/guide/ProgressDots.vue';
+import Swipe from '@/assets/js/swipe';
+
+export default {
+  name: 'OnboardingModal',
+  components: { Fluent, Modal, ProgressDots },
+  computed: {
+    first() {
+      return this.step === 1;
+    },
+    last() {
+      return this.step === this.steps.length;
+    },
+  },
+  methods: {
+    next() {
+      if (this.step < this.steps.length) {
+        this.step += 1;
+        this.step_data = this.steps[this.step - 1];
+      }
+    },
+    prev() {
+      if (this.step > 1) {
+        this.step -= 1;
+        this.step_data = this.steps[this.step - 1];
+      }
+    },
+    done() {
+      this.$store.state.onboarding.modal = false;
+      this.$emit('close');
+    },
+  },
+  mounted() {
+    this.swipe = new Swipe(this.$el, this.next, this.prev);
```

Comment:
Bringing this up out of curiosity: In Right-to-left regions, do they also swipe to the right to go to the next tile normally? I just wonder whether this is a native experience for them as well. :)

Reply:
Interesting question. That would affect the forward / backward button in FF, too?